crates/hardware/src/lib.rs

```diff
@@ -44,6 +44,24 @@ fn system76_driver(os_release: &OsRelease) -> Option<&'static str> {
     }
 }
 
+fn version(os_release: &OsRelease) -> Option<(u8, u8)> {
+    let mut components = os_release.version.split('.');
+    let major = components.next()?.parse::<u8>().ok()?;
+    let minor = components.next()?.parse::<u8>().ok()?;
+    Some((major, minor))
+}
+
+fn hp_vendor(os_release: &OsRelease) -> Option<&'static str> {
+    if &os_release.name == "Pop!_OS" && version(os_release).unwrap_or((0, 0)) >= (22, 04) {
```

Comment:
This specific line is the one I am most worried about. @ids1024 did you test that this if statement in isolation returns true on HP Dev One?